### PR TITLE
IUI 2024

### PR DESCRIPTION
**Pull request by gusongen**
### Which conference does this PR update?

- IUI 2024

### Related URL

- https://iui.acm.org/2024/

### CODE REVIEW

File: conference/HI/iui.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  dblp: iui
+  confs:
+    - year: 2024
+      id: iui24
+      link: https://iui.acm.org/2024/
+      timeline:
+        - abstract_deadline: '2023-10-02 23:59:59'
+          deadline: '2023-10-09 23:59:59'
+      timezone: AoE
+      date: May 18 - May 21, 2024
```
Comment (ViGeng) [inline conference/HI/iui.yml:14]: I think it's between 18-21 `March`?

**Review by ViGeng — CHANGES_REQUESTED** [review]
I suppose the conf time is in March?

File: conference/HI/iui.yml (hunk 1 above)
Comment (ViGeng) [reply conference/HI/iui.yml]: updated.

**Comment by gusongen** [comment]
> I suppose the conf time is in March?

You are right, thx for correction !

**Pull request merged**